packages/documentation-framework/components/tableOfContents/tableOfContents.css

```diff
@@ -12,23 +12,24 @@
   padding-block-end: var(--pf-t--global--spacer--md);
   padding-inline-start: var(--pf-t--global--spacer--md);
   padding-inline-end: 0;
-/*   box-shadow: var(--pf-v6-global--BoxShadow--lg-bottom); */
+  box-shadow: var(--pf-t--global--box-shadow--lg--bottom);
 }
 
 .ws-toc.pf-m-expanded {
-/*   box-shadow: var(--pf-v6-global--BoxShadow--sm-bottom) */
+  box-shadow: var(--pf-t--global--box-shadow--sm--bottom)
 }
 
 /* Mobile jumplinks */
 @media (max-width: 1450px) {
   .ws-toc.pf-m-expanded .pf-v6-c-jump-links__main {
     max-height: 65vh;
+    overflow-y:scroll;
   }
 
   .ws-toc .pf-v6-c-jump-links__header {
     position: sticky;
     top: 0;
-    background-color: var(--pf-t--global--background--color--primary--default);
+    background-color: var(--pf-t--global--background--color--200);
```

Comment:
If you want this to be the grey background, use `--pf-t--global--background--color--secondary--default`